### DISCUSSION TITLE
I don't receive meteor, but I receive noaa well

### DISCUSSION THREAD

**Discussion by ik5hgl** [jekhokie/raspberry-noaa-v2#789] [General]
I don't receive meteor, but I receive noaa well

He doesn't seem to get anything
Can anyone help me ?
Thank you
(hardware RTL-SDR V3)

**Below is an excerpt from the log**

05-10-2023 07:19 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : The system have enough space to store a Meteor pass on RAM
05-10-2023 07:19 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Free : 777 ; Required : 600
05-10-2023 07:19 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Direction Southbound
05-10-2023 07:19 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Starting rtl_fm record
INFO : Recording at 137.1000 MHz...
05-10-2023 07:19 /home/pi/raspberry-noaa-v2/scripts/audio_processors/meteor_record_rtl_fm.sh INFO : Recording at 137.1000 MHz...
Found 1 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001

Using device 0: Generic RTL2832U OEM
Found Rafael Micro R820T tuner
Tuner gain set to 44.50 dB.
Tuned to 137388000 Hz.
Oversampling input by: 4x.
Oversampling output by: 1x.
Buffer size: 7.11ms
Allocating 15 zero-copy buffers
Sampling at 1152000 S/s.
Output at 288000 Hz.
Signal caught, exiting!

User cancel, exiting...
05-10-2023 07:35 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Demodulation in progress (QPSK)
Input: /var/ramfs/METEOR-M-2-20231005-071936.wav, output: /home/pi/raspberry-noaa-v2/tmp/meteor/METEOR-M-2-20231005-071936.qpsk
Demodulator initialized

(  0.0%) Carrier:    +0.0 Hz, Symbol rate: 72000.0 Hz, Locked: No
(  2.6%) Carrier:  +448.2 Hz, Symbol rate: 71989.6 Hz, Locked: No
(  5.4%) Carrier: +1190.8 Hz, Symbol rate: 71985.1 Hz, Locked: No
(  8.0%) Carrier: -1437.8 Hz, Symbol rate: 71986.3 Hz, Locked: No
( 10.6%) Carrier:  -451.5 Hz, Symbol rate: 71992.4 Hz, Locked: No
( 13.3%) Carrier:  +472.5 Hz, Symbol rate: 71997.2 Hz, Locked: No
( 15.9%) Carrier:  -189.8 Hz, Symbol rate: 71995.3 Hz, Locked: No
........
05-10-2023 07:36 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Producing spectrogram
ffmpeg version 4.3.6-0+deb11u1+rpt5 Copyright (c) 2000-2023 the FFmpeg developers
  built with gcc 10 (Raspbian 10.2.1-6+rpi1)
..........
Guessed Channel Layout for Input Stream #0.0 : stereo
Input #0, wav, from '/var/ramfs/METEOR-M-2-20231005-071936.wav':
  Duration: 00:16:03.16, bitrate: 3072 kb/s
    Stream #0:0: Audio: pcm_s16le ([1][0][0][0] / 0x0001), 96000 Hz, stereo, s16, 3072 kb/s
Stream mapping:
  Stream #0:0 (pcm_s16le) -> showspectrumpic
  showspectrumpic -> Stream #0:0 (png)
Press [q] to stop, [?] for help
Output #0, image2, to '/home/pi/raspberry-noaa-v2/tmp/temp_spectrogram.png':
  Metadata:
    encoder         : Lavf58.45.100
    Stream #0:0: Video: png, rgb24, 698x928 [SAR 1:1 DAR 349:464], q=2-31, 200 kb/s, 93.75 fps, 93.75 tbn, 93.75 tbc
    Metadata:
      encoder         : Lavc58.91.100 png
frame=    0 fps=0.0 q=0.0 size=N/A time=00:00:00.00 bitrate=N/A speed=   0x    
...........
video:42kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
ffmpeg version 4.3.6-0+deb11u1+rpt5 Copyright (c) 2000-2023 the FFmpeg developers
  built with gcc 10 (Raspbian 10.2.1-6+rpi1)
.........
Input #0, png_pipe, from '/home/pi/raspberry-noaa-v2/tmp/temp_spectrogram.png':
  Duration: N/A, bitrate: N/A
    Stream #0:0: Video: png, rgb24(pc), 698x928 [SAR 1:1 DAR 349:464], 25 tbr, 25 tbn, 25 tbc
Stream mapping:
  Stream #0:0 -> #0:0 (png (native) -> png (native))
Press [q] to stop, [?] for help
[Parsed_drawtext_0 @ 0x219ab50] Using "/usr/share/fonts/truetype/ttf-bitstream-vera/Vera.ttf"
Output #0, image2, to '/srv/images/METEOR-M-2-20231005-071936-spectrogram.png':
  Metadata:
    encoder         : Lavf58.45.100
    Stream #0:0: Video: png, rgb24, 698x928 [SAR 1:1 DAR 349:464], q=2-31, 200 kb/s, 25 fps, 25 tbn, 25 tbc
    Metadata:
      encoder         : Lavc58.91.100 png
frame=    1 fps=0.0 q=-0.0 Lsize=N/A time=00:00:00.04 bitrate=N/A speed=0.083x    
video:44kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
05-10-2023 07:39 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Free memory : 703 ; Available memory : 765 ; Total RAMFS usage : 353M
05-10-2023 07:39 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Deleting audio files
05-10-2023 07:39 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Running MeteorDemod to demodulate QPSK file, rectify (spread) images, create heat map and composites and convert them to JPG
Pos:42195071 | Phase:2 | synch:84828e30 | RS: (-1, -1, -1, -1)		
Pos:44005266 | Phase:5 | synch:18fcc16c | RS: (-1, -1, -1, -1)		
Pos:54843298 | Phase:5 | synch:14ae6e68 | RS: (-1, -1, -1, -1)		
Pos:66355338 | Phase:2 | synch:2377965e | RS: (-1, -1, -1, -1)		
Pos:70984478 | Phase:1 | synch:6bd4735f | RS: (-1, -1, -1, -1)		
Pos:125639607 | Phase:5 | synch:82984858 | RS: (-1, -1, -1, -1)		
Pos:134191116 | Phase:2 | synch:12a00e5a | RS: (-1, -1, -1, -1)		
**Decoded packets:0**
**No data received, exiting...**
05-10-2023 07:45 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh ERROR : Receiver type 'rtl_fm' not valid
05-10-2023 07:45 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Producing polar graph of azimuth and elevation for pass
05-10-2023 07:45 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Producing polar graph of direction for pass
convert: unable to open image `spread_*.jpg': No such file or directory @ error/blob.c/OpenBlob/2924.
convert: no images defined `spread_*.jpg' @ error/convert.c/ConvertImageCommand/3229.
05-10-2023 07:45 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Annotating images and creating thumbnails
Loading page (1/2)
[>                                                           ] 0%
[======>                                                     ] 10%
[=========>                                                  ] 15%
[=============>                                              ] 23%
[============================================================] 100%
Rendering (2/2)                                                    
[>                                                           ] 0%
[===============>                                            ] 25%
[============================================================] 100%
Done                                                               
convert: unable to open image `*.jpg': No such file or directory @ error/blob.c/OpenBlob/2924.
convert: image sequence is required `-composite' @ error/mogrify.c/MogrifyImageList/8003.
convert: no images defined `*.jpg' @ error/convert.c/ConvertImageCommand/3229.
convert: unable to open image `*.jpg': No such file or directory @ error/blob.c/OpenBlob/2924.
convert: no images defined `*-thumb-122-rectified.jpg' @ error/convert.c/ConvertImageCommand/3229.
05-10-2023 07:45 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh ERROR : Did not get a successful .bmp image - stopping processing
05-10-2023 07:45 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Pushing images to Discord
05-10-2023 07:45 /home/pi/raspberry-noaa-v2/scripts/receive_meteor.sh INFO : Total processing time: 00:26.52

**Comment by Itsisme** [comment]
Hai,

I have the same effect NOAA working well METEO M2 3 nothing happens.
I saw in the TLE data METEO m2 and METEO m2 3 maybe its confused wit its data ??

Would be great if that would be fixed by some one...

Chrs..
